core/app/modules/kit-library/assets/js/components/page-loader.scss

```
@@ -0,0 +1,9 @@
+.e-kit-library__page-loader {
+	width: 100%;
+	height: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
```

```suggestion
	display: grid;
	place-items: center;
```

I mean, just if you want... 😝

---

Learn something new.
Done.